plugins/config_manager.py

```diff
@@ -60,6 +60,20 @@ def configure(self):
         demands = self.cli.demands
         demands.available_repos = True
 
+        # if no argument was passed then error
+        if (not (self.opts.crepo != [] or
+                 self.opts.add_repo != [] or
+                 self.opts.save or
+                 self.opts.dump or
+                 self.opts.dump_variables or
+                 self.opts.set_disabled or
+                 self.opts.set_enabled) ):
+            self.cli.optparser.error(_("one of the arguments {} is required")
```

Comment:
This format is not very readable. I'd rather put the list of arguments at the very end, such as: "one of the following arguments is required: {}".